src/mono/wasm/testassets/WasmBasicTestApp/ResourceLibrary/ResourceAccessor.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Globalization;
+using System.Threading.Tasks;
+using System.Resources;
+
+namespace ResourceLibrary;
+
+public static class ResourceAccessor
+{
+    public static void Read(Action<string> testOuput)
+    {
+        var rm = new ResourceManager("ResourceLibrary.words", typeof(ResourceAccessor).Assembly);
+        testOuput("default: " + rm.GetString("hello", CultureInfo.CurrentCulture));
+        testOuput("es-ES without satellite: " + rm.GetString("hello", new CultureInfo("es-ES")));
```

Comment:
This is not 100% true, 1st call is `without`, 2nd call should be `with`. Should this method have another parameter `bool isSateliteAssemlbyLoaded`? This broke some tests.

Reply:
Thamk you!